cms/templates/js/publish-xblock.underscore

```diff
@@ -19,43 +19,43 @@ if (visibilityState === 'live') {
 
 var visibleToStaffOnly = visibilityState === 'staff_only';
 %>
-<div class="bit-publishing <%= visibilityClass %> <% if (releaseDate) { %>is-scheduled<% } %>">
-    <h3 class="bar-mod-title pub-status"><span class="sr"><%= gettext("Publishing Status") %></span>
-        <%= title %>
+<div class="bit-publishing <%- visibilityClass %> <% if (releaseDate) { %>is-scheduled<% } %>">
+    <h3 class="bar-mod-title pub-status"><span class="sr"><%- gettext("Publishing Status") %></span>
+        <%- title %>
     </h3>
 
     <div class="wrapper-last-draft bar-mod-content">
         <p class="copy meta">
             <% if (hasChanges && editedOn && editedBy) {
                 var message = gettext("Draft saved on %(last_saved_date)s by %(edit_username)s") %>
-                <%= interpolate(message, {
-                    last_saved_date: '<span class="date">' + editedOn + '</span>',
-                    edit_username: '<span class="user">' + editedBy + '</span>' }, true) %>
+                <%= interpolate(_.escape(message), {
+                    last_saved_date: '<span class="date">' + _.escape(editedOn) + '</span>',
+                    edit_username: '<span class="user">' + _.escape(editedBy) + '</span>' }, true) %>
```

Comment:
Hi @andy-armstrong.  For when you return...
Above is what this code currently looks like with proper escaping.  This is where it would be great to end up with something that reads more like the Mako example and `HTML()` wrapper methods here:
http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#style-guidelines

Additionally, it would be great if we could use `<%-` and it could be smart enough to know if it is getting a special object that was already escaped.